docs/concepts/vega-chain.md

```diff
@@ -143,37 +143,113 @@ Read more: [Risks of over-staked validators](/docs/concepts/vega-chain#too-much-
+Read more: [How a validating node's performance is determined](/docs/concepts/vega-chain#validating-node-performance)
 
-## Validator nodes
-The Vega network is operated by a number of independent validators, who each run a node. Validator nodes are responsible for agreeing on the order of transactions and creating new blocks so that all nodes can agree on the state of the network. 
+## Validating nodes
+The Vega network is operated by a number of independent validators, who each run a node.
 
-If a validator's stake or performance is sub-par, their validator score will be lowered, and that validator's node will be chosen less frequently to propose a block. (This can also affect the rewards they and their nominators receive.) Vega feeds the voting power of each validator node to the Tendermint consensus algorithm. 
+There are two types of validating nodes: consensus validators and standby validators.
 
-In this version of restricted mainnet, validators will not lose stake or rewards if they have a temporary interruption of service.
+## Consensus validating nodes
+Consensus validating nodes are responsible for agreeing on the order of transactions and creating new blocks so that all nodes can agree on the state of the network.
 
-### Validator node performance
-A validator node's performance is expressed through a validator score, and in the future, a performance score as well. 
+They receive rewards based on having enough (but not too much) self-stake, as well as how many tokenholders nominate them. The tokenholders who nominate them also receive a cut of the rewards.
 
-The validator score is calculated for each epoch, based on how much stake a validator has as well as other factors including the total number of validators and the optimal stake. 
+If a consensus validator's stake or performance is subpar, their validator score will be lowered, and that validator's node will be chosen less frequently to propose a block, because Vega feeds the voting power of each validating node to the Tendermint consensus algorithm.
 
-If, at the end of an epoch, a validator does not have sufficient stake self-nominated or has overall too much stake, then their validator score will be lowered, which can impact the rewards a validator and its nominators receive. 
+This can also affect the rewards they and their nominators receive.
+If a consensus validating node stops validating, or performs poorly, then a standby validator can replace it.
+
+### How consensus validators are chosen
+Consensus validators are chosen based on a range of variables, including the validator scores, and the number of available slots. If there are no empty slots, at most one validator can be changed per epoch.
+
+If a standby validator has a better overall performance than an existing consensus validator, then a standby validator can be promoted to replace a consensus validator.
```

Comment:
_better overall performance_ - for clarity, it's not just performance, it's also stake.